package.json

```diff
@@ -14,14 +14,17 @@
       "react-dom": "^18.2.0"
    },
    "devDependencies": {
+      "@playwright/test": "^1.44.0",
       "@types/react": "^18.2.74",
       "@types/react-dom": "^18.2.24",
       "@vitejs/plugin-react": "^4.2.1",
       "@wixc3/react-board": "^2.4.1",
+      "chai": "^5.1.0",
+      "mocha": "^10.4.0",
+      "prettier": "3.1.0",
       "sass": "^1.74.1",
       "typescript": "^5.4.3",
       "vite": "^5.2.8",
-      "vite-plugin-svgr": "^4.2.0",
-      "prettier": "3.1.0"
```

Comment:
Why did you remove prettier from the project? and how is it related to this PR?

Reply:
I didn’t. For some reason, the installation of Playwright moved it a few lines up.